example_project_app/plugins.py

```diff
@@ -275,3 +275,25 @@ def perform_project_modify(
                     action=action.lower(),
                 ),
             )
+
+    def validate_form_app_settings(self, app_settings, project=None, user=None):
+        """Example implementation for app setting validation plugin API"""
+        errors = {}
+        for setting_name in app_settings:
+            if (
+                app_settings[setting_name].get('scope')
```

Comment:
Of course, definitions such as scope will not be available in this value now because you are simply providing the value. But that's ok, we don't necessarily have to validate against those here. Just come up with some rules for the example project app values and check those in test :)